pegasus/sites.v3/code.org/public/js/volunteer-local.js

```diff
@@ -285,6 +286,45 @@ function contactVolunteer() {
 
   return false;
 }
+
+function processResponse(data) {
+  $('#error-message').hide();
+  $('#success-message').show();
+}
+
+function processError(data) {
+  $('.has-error').removeClass('has-error');
+
+  var errors = Object.keys(data.responseJSON);
+  var errors_count = errors.length;
+
+  for (var i = 0; i < errors_count; ++i) {
+    var error_id = '#volunteer-contact-' + errors[i].replace(/_/g, '-');
+    error_id = error_id.replace(/-[sb]s?$/, '');
+    $(error_id).parents('.form-group').addClass('has-error');
+  }
+
+  var error = '<font color="#a94442">An error occurred. All fields are required. Please check that all fields have been filled out properly.</font>';
+  $('#error-message').html(error).hide().fadeTo("normal", 1);
```

Comment:
just US/English volunteers or do we need a way to translate these error messages?

Reply:
We haven't localised any of the experience at /volunteer or /volunteer/local.

Reply:
Actually /volunteer and /volunteer/local do have a lot localized already, but it's not a goal. Wouldn't worry about this for this PR.